### PR TITLE
vsmile_cart.xml: Add UK versions of several V.Smile games

### PR DESCRIPTION
**Pull request by clawgrip**
New WORKING software list additions
----------------------------------
Nick Jr. Dora the Explorer - Dora's Fix-it Adventure (UK, 2005), Nick Jr. Go Diego Go! - Save the Animal Families! (UK), Disney's Mickey Mouse - Mickey's Magical Adventure (UK), Scooby-Doo! - Funland Frenzy (UK, 2004), Marvel Spider-Man & Friends - Secret Missions (UK), Disney's Winnie de Pooh - The Honey Hunt (UK), Zayzoo - My Alien Classmate (UK), Disney/Pixar Cars - Rev It Up in Radiator Springs (UK, alt), Alphabet Park Adventure (UK), Disney's Aladdin - Aladdin's Wonders of the World (UK) [TeamEurope]

### CODE REVIEW

File: hash/vsmile_cart.xml [hunk 1]
```diff
@@ -629,6 +630,21 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
...
+		<part name="cart" interface="vsmile_cart">
+			<feature name="slot" value="vsmile_rom" />
+			<feature name="cart_type" value="lilac" />
+			<feature name="u1" value="" />
+			<dataarea name="rom" size="8388608">
+				<rom name="52-92203- Disney Aladdin - Wonders of the World (UK).bin" size="8388608" crc="244eb37a" sha1="c8131f488e563ef1a7f045ef3601670d02ca404e" />
+			</dataarea>
+		</part>
+	</software>
+
```
Comment (cuavas) [inline hash/vsmile_cart.xml:647]: Should this be the new parent?  We usually go for English-language parents.

Comment (clawgrip) [reply hash/vsmile_cart.xml]: Yes!
I was thinking on using the VTech precedence (USA -> Italy -> Netherlands -> UK -> Germany...), but makes more sense to follow the usual "English first".

Thanks!!!

File: hash/vsmile_cart.xml [hunk 2]
```diff
@@ -165,6 +165,7 @@ Regular game cartridges
 |        | 80-092062(NL)     | Winnie de Poeh - De wonderwereld van Poeh                                                      |
 |   XX   | 80-092062-123(NL) | Winnie de Poeh - De wonderwereld van Poeh (rev. 123, labeled as "My Friends Tigger and Pooh", 52-092062(NL) on back label)
 |        | 80-092063(UK)     | Winnie The Pooh: The Honey Hunt                                                                |
+|   XX   | 80-092063-103(UK) | Winnie The Pooh: The Honey Hunt (52-092063-103(UK) on back label                               |
 |   XX   | 80-092064(GE)     | Pooh - Die Honigjagd                                                                           |
```
Comment (cuavas) [inline hash/vsmile_cart.xml:169]: Are the parentheses in this comment supposed to balance, or is the opening parentheses in “(52-” part of what’s on the back label?

File: hash/vsmile_cart.xml [hunk 3]
```diff
@@ -165,12 +165,13 @@ Regular game cartridges
 |        | 80-092062(NL)     | Winnie de Poeh - De wonderwereld van Poeh                                                      |
 |   XX   | 80-092062-123(NL) | Winnie de Poeh - De wonderwereld van Poeh (rev. 123, labeled as "My Friends Tigger and Pooh", 52-092062(NL) on back label)
 |        | 80-092063(UK)     | Winnie The Pooh: The Honey Hunt                                                                |
+|   XX   | 80-092063-103(UK) | Winnie The Pooh: The Honey Hunt (52-092063-103(UK)) on back label                              |
```
Comment (cuavas) [inline hash/vsmile_cart.xml:168]: The format isn’t the same between these where “on back label” is inside the parentheses for the NL version, but outside the parentheses for the UK version.

Comment (clawgrip) [reply hash/vsmile_cart.xml]: Arggggg... Yes, sorry.

**Pull request merged**